### DISCUSSION TITLE
Can you explain the application scenario of the new Artisan PID SV synchronization

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1749] [General]
The input source selection in the modbus pid control dialog cannot be modified，The input source can only be changed in the ports configuratio modbus dialog box.
![974d11ea92b142222bc9dc5af875b38](https://github.com/user-attachments/assets/8d62bbc9-5b1a-4b4e-a440-361d7e16e8f1)

**Comment by MAKOMO** [comment]
Indeed, for the external MODBUS PID control, the PID Input cannot be set from within Artisan as this is set in the external PLC PID. However, that PID Input selection here is used to decide which curve to follow in Background Follow mode. 

That new Set Value "Sync" selector allows to interpret one of the curves data as (external) SV signal and move the SV slider with the data received.
